app/src/main/java/ru/semper_viventem/pixel4scanner/ShareUtils.kt

```diff
@@ -53,6 +55,20 @@ private fun saveAsFile(activity: Activity, bitmap: Bitmap): Uri {
     )
 }
 
+private fun saveAsFile2(activity: Activity, text: String): Uri {
```

Comment:
What do you think if we will call this method just `saveAsFile`? Without `2`. This method will overloaded previous method but with String argument.